### ISSUE TITLE
"Save All" not working

### ISSUE THREAD

**Issue by emilianos** [atom/atom#16384]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

When having several unsaved tabs open, using the shortcut "alt-cmd-s" doesn't work or do anything.
Going into the "File" menu and selecting "Save All" still works.

### Steps to Reproduce

1. Open and edit more than one file in different tabs
2. Try to save everything using the shortcut: "alt-cmd-s"
3. nothing happens..

**Expected behavior:**
Should save all changes

**Actual behavior:**
Nothing happens

**Reproduces how often:**
Always failing

### Versions

Atom    : 1.23.0
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0

apm  1.18.11
npm  3.10.10
node 6.9.5 x64
atom 1.23.0
python 2.7.10
git 2.10.0

MacOS High Sierra 10.13.1

### Additional Information

I think it has to do with the use of the "alt" key, remapping the shortcut to different key combination works while not using "alt" in it.
Probably has to do with the OS.

![screen shot 2017-12-14 at 11 10 41](https://user-images.githubusercontent.com/4334689/33996267-7c0801d2-e0bf-11e7-9834-a3ea1d80f7c0.png)

**Comment by rsese** [comment]
Thanks for the report!  I'm unable to reproduce on macOS 10.12.6 and Atom 1.23.1 - can you share a screenshot of the [keybinding resolver](http://flight-manual.atom.io/hacking-atom/sections/debugging/#check-your-keybindings) when you try to save all with <kbd>alt-cmd-s</kbd>?

**Comment by emilianos** [comment]
Thank you for the quick reply!

Yesterday I was trying the Keybinding resolver and it wasn't doing anything for save-all, but it did for other combinations.
So in order to capture that I thought of doing a video this morning. The funny thing is this time save-all worked!

I'm not sure what could have been wrong, I have had this problem for a couple of months now and yesterday I did different changes and then restarted my machine, so maybe that had something to do with it.

Another thing I did was to set the shortcut in the mac preferences. I'm going to remove it and see how it goes from there.

![screen shot 2017-12-15 at 08 37 12](https://user-images.githubusercontent.com/4334689/34040687-8d362b46-e173-11e7-99ca-09f52b634a88.png)
